test/services/query-options.test.js

```diff
@@ -4,33 +4,54 @@ import QueryOptions from '../../src/services/query-options';
 
 describe('services > query-options', () => {
   describe('order', () => {
-    const model = {
-      name: 'actor',
-      unscoped: () => model,
-      sequelize: { constructor: Sequelize },
-      associations: {},
+    const buildModelMock = (dialect) => {
+      const sequelize = new Sequelize({ dialect });
```

Comment:
[Suggestion] I think a small comment here could be useful about the fact the you are instanciating a sequelize connection without the need of the database.
I had hard time figuring that:
- `new Sequelize()` does not connect to an actual database here
- That you are mocking the actual method of sequelize call (`destroy`) here. Without the mock, it would throw